image.yaml

```diff
@@ -2,7 +2,7 @@ schema_version: 1
 name: "jboss-dataservices/datagrid-online-services"
 description: "Red Hat JBoss Data Grid Services"
 version: "1.0"
-from: "jboss-datagrid-7/datagrid72-openshift:dev"
+from: "docker-registry.engineering.redhat.com/jboss-datagrid-7/datagrid72-openshift:dev"
```

Comment:
This is something that can stay for development but we need to get rid of it in longer run. With this piece, we won't be able to release this through OSBS.

Could you please contact CE folks and ask them what is the configuration they are using?

Reply:
Yes. I'll talk to them. This was I was just trying to workaround my local problems with Docker. I'll remove this when sending a real PR.

Reply:
I spoke with xPaaS QE and they're using the RedHat's fork of Docker so they specify multiple registries via --add-registry option. But I don't have this option in my installation.